src/geo/map/cartodb-layer.js

```diff
@@ -31,6 +45,17 @@ var CartoDBLayer = LayerModelBase.extend({
     return this.get('visible');
   },
 
+  getGeometryType: function () {
```

Comment:
We could but I don't think it should. cartodb.js does not need to know anything about the geometry type, that's something tied to the editor not cartodb.js